Problem: make proto-all not working

make proto-all : proto-lint error 
make  update-swagger-docs  : working fine
```
...
{"path":"proto/supply/v1/query.proto","start_line":18,"start_column":3,"end_line":20,"end_column":4,"type":"RPC_REQUEST_RESPONSE_UNIQUE","message":"\".chainmain.supply.v1.SupplyResponse\" is used as the request or response type for multiple RPCs."}
{"path":"proto/supply/v1/query.proto","start_line":18,"start_column":44,"end_line":18,"end_column":58,"type":"RPC_RESPONSE_STANDARD_NAME","message":"RPC response type \"SupplyResponse\" should be named \"LiquidSupplyResponse\" or \"QueryLiquidSupplyResponse\"."}
make: *** [Makefile:246: proto-lint] error 100

```

--- a/Makefile
+++ b/Makefile
@@ -223,5 +223,11 @@ release:
 ###                              Documentation                              ###
 ###############################################################################
 
+# generate api swagger document
 document:
 	make all -f MakefileDoc
+
+# generate protobuf files
+# ./proto -> ./x
+proto-all:
+	make proto-all -f MakefileDoc

--- a/MakefileDoc
+++ b/MakefileDoc
@@ -28,10 +28,7 @@ proto-format:
 	@echo "Formatting Protobuf files"
 	find ./ -not -path "./third_party/*" -name *.proto -exec clang-format -i {} \;
 
-proto-lint:
-	@docker run --rm -v $(CURDIR):/workspace --workdir /workspace bufbuild/buf lint --error-format=json
-
-proto-all: proto-format proto-lint proto-gen
+proto-all: proto-format proto-gen
 
 update-swagger-docs: statik
 	$(BINDIR)/statik -src=app/docs/swagger-ui -dest=app/docs -f -m

--- a/scripts/protocgen
+++ b/scripts/protocgen
@@ -7,7 +7,10 @@
 set -eo pipefail
 
 proto_dirs=$(find ./proto -path -prune -o -name '*.proto' -print0 | xargs -0 -n1 dirname | sort | uniq)
+echo "Generate protobuf files from ./proto to ./x"
 for dir in $proto_dirs; do
+  echo "processing "${dir}
+# buf is enhanced protobuf generator
   buf protoc \
   -I "proto" \
   -I "third_party/cosmos-sdk/proto" \

